Packs/Anomali_Enterprise/Integrations/Anomali_Enterprise/Anomali_Enterprise.py

```diff
@@ -0,0 +1,294 @@
+from typing import Dict
+
+import urllib3
+from CommonServerPython import *
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+VENDOR_NAME = 'Anomali Enterprise'
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """
+    Client to use in the Anomali Enterprise integration. Overrides BaseClient
+    """
+
+    def __init__(self, server_url: str, username: str, password: str, verify: bool, proxy: bool):
+        super().__init__(base_url=server_url, verify=verify, proxy=proxy)
+        self._username = username
+        self._password = password
+        self._headers = {'Content-Type': 'application/json'}
```

Comment:
why you create new contractor if you can use the existing one. And pass the auth=(username,password), headers=headers
to it

Reply:
the username/password have to go in the data for and not in the headers for the start_search_job_request, domain_request methods, i've checked that.
anyhow, i have moved to create the `_headers` only once as you suggested, as it is better practice.